drivers/hmis/app/graphql/mutations/update_relationship_to_ho_h.rb

```diff
@@ -13,82 +13,24 @@ class UpdateRelationshipToHoH < BaseMutation
 
     field :enrollment, Types::HmisSchema::Enrollment, null: true
 
-    def self.incomplete_hoh_message
-      'Selected HoH has an incomplete enrollment.'
-    end
-
-    def self.exited_hoh_message
-      'Selected HoH is exited.'
-    end
-
-    def self.child_hoh_message
-      'Selected HoH is a child.'
-    end
-
-    def self.change_hoh_message(old_hoh, new_hoh)
-      old_hoh_name = old_hoh&.brief_name
-      new_hoh_name = new_hoh&.brief_name
-      if old_hoh_name.present?
-        "Head of Household will change from #{old_hoh_name} to #{new_hoh_name}."
-      else
-        "#{new_hoh_name} will be the Head of Household."
-      end
-    end
-
     def resolve(enrollment_id:, enrollment_lock_version: nil, relationship_to_ho_h:, confirmed: false)
       enrollment = Hmis::Hud::Enrollment.viewable_by(current_user).find_by(id: enrollment_id)
-      raise 'Not found' unless enrollment.present?
-      raise 'Access denied' unless current_user.permissions_for?(enrollment, :can_edit_enrollments)
+      access_denied! unless enrollment.present?
+      access_denied! unless current_user.permissions_for?(enrollment, :can_edit_enrollments)
```

Comment:
present is not needed here, this will be nil if the record is not found. Maybe golf down?
```suggestion
      access_denied! unless enrollment && current_user.permissions_for?(enrollment, :can_edit_enrollments)
```